pkg/agent/keys_darwin.go

```diff
@@ -0,0 +1,63 @@
+//go:build darwin
+// +build darwin
+
+package agent
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/krypto/pkg/secureenclave"
+	"go.etcd.io/bbolt"
+)
+
+func setupHardwareKeys(logger log.Logger, db *bbolt.DB) error {
+	_, pubData, err := fetchKeyData(db)
+	if err != nil {
+		return err
+	}
+
+	if pubData == nil {
+		level.Info(logger).Log("Generating new keys")
+		pub, err := secureenclave.CreateKey()
+		if err != nil {
+			return fmt.Errorf("creating key: %w", err)
+		}
+
+		if err := storeKeyData(db, nil, ecdsaToRaw(pub)); err != nil {
+			return fmt.Errorf("storing key: %w", err)
+		}
+
+		k, err := secureenclave.New(*pub)
+		if err != nil {
+			return fmt.Errorf("creating secureenclave signer:, from new key %w", err)
```

Comment:
`:,` -- typo or missing another arg we wanted to include in this error message?

Reply:
discussing deletion below